src/content/app/genome-browser/components/track-panel/components/track-panel-list/TrackPanelList.test.tsx

```diff
@@ -87,5 +115,31 @@ describe('<TrackPanelList />', () => {
       );
       expect(container.querySelector('.mainTrackItem')).toBeFalsy();
     });
+
+    it('renders regular tracks', async () => {
+      const { container } = renderComponent();
+
+      await waitFor(() => {
+        expect(
+          container.querySelectorAll('.trackPanelRegularItem').length
```

Comment:
Same with the formatting here as well. Wonder why prettier didn't format these during the commit process.

Reply:
As we discussed in the daily scrum this was just in github. The actual code in a text editor looks alright.